tests/client/amo/components/TestAddonDetail.js

```diff
@@ -139,6 +139,23 @@ describe('AddonDetail', () => {
                    'About this extension');
   });
 
+  it('uses the summary as the description if no description exists', () => {
+    const addon = { ...fakeAddon };
```

Comment:
This is the right thing to do but you reminded me that we should add `Object.freeze()` to both `fakeAddon` and `fakeReview` just like [signedInApiState](https://github.com/mozilla/addons-frontend/blob/master/tests/client/amo/helpers.js#L55). Could you add this in your patch?